app/models/messages.py

```diff
@@ -1,14 +1,42 @@
-from app.models import *
+from datetime import datetime
 
+from sqlalchemy import Column, DateTime, Enum, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
 
-class Messages(Base):
-    __tablename__ = "messages"
+from app.models import Base
+
+
+class ChatRoom(Base):

```

Comment:
It's a `ChatRoom` but it's in the `messages.py` 🤔. I would leave only messages here `User message`, maybe `System Message`, and move the chat room  with matching to `chat_room.py`.